crates/red_knot_python_semantic/src/types/unpacker.rs

```diff
@@ -63,6 +63,11 @@ impl<'db> Unpacker<'db> {
             });
         }
 
+        if value.is_context_manager() {
```

Comment:
I almost missed this because it's not a match and the compiler didn't catch me. Would folks be okay with my refactoring it?

Reply:
Totally.